drivers/remote/driver.go

```diff
@@ -247,6 +247,33 @@ func (d *driver) Leave(nid, eid string) error {
 	return d.call("Leave", leave, &api.LeaveResponse{})
 }
 
+// ProgramExternalConnectivity is invoked to program the rules to allow external connectivity for the endpoint.
+func (d *driver) ProgramExternalConnectivity(nid, eid string, options map[string]interface{}) error {
+	data := &api.ProgramExternalConnectivityRequest{
+		NetworkID:  nid,
+		EndpointID: eid,
+		Options:    options,
+	}
+	err := d.call("ProgramExternalConnectivity", data, &api.ProgramExternalConnectivityResponse{})
+	if err != nil {
```

Comment:
The fact is I did not want to add a `supports external connectivity` capability only to cover the deprecation window.
But it's true this change will mask real error along not supported errors.

Maybe it instead makes sense to add the new capability flag. I can think of network drivers which cannot provide the external connectivity. This can be a generic case.
Current remote drivers are already queried for capabilities, therefore no changes, the default value for the new cap would be false, which is what we want.

I will make the change. Thanks.